README.md

```diff
@@ -42,7 +42,7 @@ Conceptually, one can workaround the twin size limitation by placing the larger
 
 ### Twin Contract 
 
-The structure of the twin's JSON payload represents a contract between the device client and the backend solution. In this example, we are defining the following properties in the twin.
+The structure of the twin's JSON payload represents a contract between the device client and the backend solution. In this example, we are defining the following properties in the twin. You will add the `desired` and `reported` properties for the device via the Azure Portal or [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/iot/hub/device-twin?view=azure-cli-latest).
```

Comment:
Can you update this with a note that the cloud services set the "desired state" and that the device or edge is responsible for updating the "reported state"

Reply:
Added a note describing a device twin's desired and reported properties.